lib/seattleflu/id3c/cli/command/etl/redcap_map.py

```diff
@@ -36,16 +36,17 @@ def map_vaccine(vaccine_response: str) -> Optional[bool]:
     (https://www.hl7.org/fhir/valueset-immunization-status.html)
     """
     vaccine_map = {
-        'Yes': True,
-        'No': False,
-        'Do not know': None,
+        'yes': True,
```

Comment:
> 

Good catch! Those have both been in place for a while, but looks like the one in `redcap.py` is only used within that module, in the `create_vaccine_item` function. It could probably import from `redcap_map.py` instead. All the other ETLs seem to use that one.

Reply:
With that said, since those two functions already existed, I'm inclined to leave them both and just be more explicit with the imports. It can be dealt with in a separate PR.

Reply:
Gotcha thanks, that makes sense. Thanks for checking it out.